File: 백준/Bronze/5086. 배수와 약수/배수와 약수.c
#include<stdio.h>
    int main()
    {
        int a, b;
        do
        {
            scanf("%d %d", &a, &b);
            if (b + a == 0)
                break;
            else if (b % a == 0)
                printf("factor\n");
            else if (a % b == 0)
                printf("multiple\n");
            else
                printf("neither\n");
        } while (1);
        return 0;
    }

File: 백준/Bronze/5086. 배수와 약수/배수와 약수.kt
import java.util.Scanner
fun main(){
    val sc = Scanner(System.`in`)
    while(true){
        var a = sc.nextInt()
        var b = sc.nextInt()
        if(a == 0 && b == 0){
            break;
        }
        else{
            var s = when{
            b%a == 0 -> "factor"
            a%b == 0 -> "multiple"
            else -> "neither"
            }
            println(s)
        }
    }
}